File: golang/loop.go
package kubeless

import (
	"fmt"
	"github.com/kubeless/kubeless/pkg/functions"
	"github.com/sirupsen/logrus"
	"time"
)

// Hello sample function with dependencies
func Hello(event functions.Event, context functions.Context) (string, error) {
	logrus.Info(event)
	fmt.Println(time.Now())
	sum := 0
	for i := 0; i < 1000000; i++ {
		sum++
	}
	fmt.Println(sum)
	fmt.Println(time.Now())
	return "Hello world!", nil
}

File: python/loop.py
import flask
import datetime as dt
def hello(event, context):
    print(flask)
    print(dt.datetime.now())
    sum = 0
    for i in range(0, 1000000):
        sum += 1
    print(sum)
    print(dt.datetime.now())
    return "Hello world!"
